mkosi/__init__.py

```diff
@@ -311,25 +405,77 @@ def gpt_root_native(arch: Optional[str], usr_only: bool = False) -> GPTRootTypeT
         arch = platform.machine()
 
     if usr_only:
-        if arch in ("i386", "i486", "i586", "i686"):
+        if arch == "alpha":
+            return GPTRootTypeTriplet(GPT_USR_ALPHA, GPT_USR_ALPHA_VERITY, GPT_USR_ALPHA_VERITY_SIG)
+        elif arch == "arc":
+            return GPTRootTypeTriplet(GPT_USR_ARC, GPT_USR_ARC_VERITY, GPT_USR_ARC_VERITY_SIG)
+        elif arch == "armv7l":
```

Comment:
there are about ~500 other names to match here... see architecture.c over in systemd

Reply:
Maybe `arch.startswith("armv")` ?
Or `re.match(r'armv[4-8]', arch)` ?

Reply:
Went with `startswith` so we don't have to modify if they start inventing new numbers